index.html

```diff
@@ -176,7 +176,34 @@ <h3 class="contact">LET'S CONNECT</h3>
                     <i class="fa fa-angle-right fa-lg angle-arrow" aria-hidden="true"></i>
                   </li>
               </ul> 
-              <!-- Categories end -->         
+              <!-- Categories end -->
+                
+              <section class="contact-me">           
+                <h2>Contact me</h2>
+                <p class="form-text">
+                     If you have an application you are interested in developing,
+                    a feature that you need built or a project that needs coding.
+                    I’d love to help with it 
+                </p>
+                <form action="https://formspree.io/f/xayvqwwl" method="POST">                
+                    <input type="text" id="name" minlength="30" name="user_name" placeholder="Abenezer" required>                   
```

Comment:
- [ ] **Required** The name input doesn't accept a name less than 30 characters because you have use `minlength="30"`. Kindly change this to `maxlength="30"`

Reply:
Ok I will change.